test/cli/integration_test.sh

```diff
@@ -20,6 +20,20 @@ function wait_for_resource() {
     done
 }
 
+function cleanup_error() {
```

Comment:
Can we simplify this with smth. like: https://unix.stackexchange.com/questions/124127/kill-all-descendant-processes ?